README.md

```diff
@@ -1,5 +1,33 @@
 # Build scripts for tor binary
 
+### GPG keys
+
+GPG keyservers are known to be flaky so we include the keys in the repo:
+
+1. Tor: `$gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 2133BC600AB133E1D826D173FE43009C4607B1FB`
+```
+$ gpg --fingerprint 2133BC600AB133E1D826D173FE43009C4607B1FB
+pub   rsa4096 2016-09-21 [C] [expires: 2020-09-16]
+      2133 BC60 0AB1 33E1 D826  D173 FE43 009C 4607 B1FB
+uid           [ unknown] Nick Mathewson <nickm@alum.mit.edu>
+uid           [ unknown] Nick Mathewson <nickm@wangafu.net>
+uid           [ unknown] Nick Mathewson <nickm@freehaven.net>
+uid           [ unknown] Nick Mathewson <nickm@torproject.org>
+sub   rsa4096 2016-09-23 [S] [expires: 2020-09-16]
+sub   rsa4096 2016-09-23 [E] [expires: 2020-09-16]
+```
+
+2. Libevent: `$gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 9E3AC83A27974B84D1B3401DB86086848EF8686D`
+```
+$ gpg --fingerprint 9E3AC83A27974B84D1B3401DB86086848EF8686D
+pub   rsa2048 2010-06-10 [SC]
+      9E3A C83A 2797 4B84 D1B3  401D B860 8684 8EF8 686D
+uid           [ unknown] Azat Khuzhin <a3at.mail@gmail.com>
+uid           [ unknown] Azat Khuzhin <bin@azat.sh>
+uid           [ unknown] Azat Khuzhin <azat@libevent.org>
+sub   rsa2048 2010-06-10 [E]
+```
+
```

Comment:
Can you say how the files gpg-keys/foo.gpg are constructed?  (This just imports them into your gpghome.)

Reply:
done!